dependency_support/systemlibs/llvm.BUILD

```diff
@@ -0,0 +1,182 @@
+# Copyright 2020 The XLS Authors
```

Comment:
Instead of keeping all of the system library BUILD files in the `systemlibs` directory, could they be put in the directory for each dependency instead?

The corresponding file for the Bazel-managed LLVM is `dependency_support/llvm/bundled.BUILD.bazel`. Could this file for example be `dependency_support/llvm/systemlib.BUILD.bazel`?

This comment applies to all other dependency-specific files that are currently under `dependency_support/systemlibs/`.